### ISSUE TITLE
[Development] Mismatch in MYSQL Driver Versions

### ISSUE THREAD

**Issue by kadeksuryam** [ia-toki/judgels#534]
In the [Developer's Guide](https://github.com/ia-toki/judgels/wiki/Dev's-Guide:-Running-from-source), the prerequisite is to use MYSQL 8. 

However, in the [`gradle.properties`](https://github.com/ia-toki/judgels/blob/master/judgels-backends/gradle.properties#L23) file, the `mysqlConnectorJavaVersion` is set to version `5.1.44`. 

This mismatch prevents the execution of the database migration process.

**Comment by fushar** [comment]
Interesting, we never had any problem with the version, even with MySQL 8.

What is the error message?

**Comment by kadeksuryam** [comment]
Thank you. As a background, I'm using this Docker Compose configuration to provision a local MySQL database:
![image](https://github.com/ia-toki/judgels/assets/21070615/ca350237-6b5f-409d-8752-40c4aab38bc9)

Next, I tried to follow the Dev's Guide. After updating the database credentials in var/conf/judgels-server.yml, I attempted to run the database migration. That's when this error occurred:

![image](https://github.com/ia-toki/judgels/assets/21070615/709c7713-2d6f-4669-86fd-08de4202b72c)

Then, I proceeded to upgrade the connector version, and the problem was resolved.

It's possible that this issue arose due to my use/misconfig of docker compose. If this problem doesn't cause any issues for others, I will closing it.
